packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/SampleAppCpp.vcxproj

```diff
@@ -111,7 +111,7 @@
       <!--Temporarily disable cppwinrt heap enforcement to work around xaml compiler generated std::shared_ptr use -->
       <AdditionalOptions Condition="'$(CppWinRTHeapEnforcement)'==''">/DWINRT_NO_MAKE_DETECTION %(AdditionalOptions)</AdditionalOptions>
       <DisableSpecificWarnings>%(DisableSpecificWarnings)</DisableSpecificWarnings>
-      <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);$(ReactNativeWindowsDir)include</AdditionalIncludeDirectories>
```

Comment:
>;$(ReactNativeWindowsDir)includ [](start = 67, length = 31)

It would be nice if we could avoid doing it. It will make it difficult for us to implement the DLL based deployment.

Reply:
This folder has too many files which we want to remove in future. If user code starts to depend on it. It will be virtually impossible to switch to the DLL based deployment.

---
In reply to: [419827879](https://github.com/microsoft/react-native-windows/pull/4766#discussion_r419827879) [](ancestors = 419827879)